yt_idefix/data_structures.py

```diff
@@ -318,6 +410,19 @@ def _parse_parameter_file(self):
         self._parse_definitions_header()
         self._setup_geometry()
 
+    def _parse_geometry(self, geom: str):
```

Comment:
this could be a `@staticmethod`, since the `self` argument isn't used

Reply:
Okay! I'll change that.

Reply:
I have made the function static